test/driver.ts

```diff
@@ -287,6 +307,11 @@ export class Participant {
     async unregisterAsValidator() {
         return await this.validatorsRegistration.unregisterValidator({from: this.address});
     }
+
+    gasUsed(): number {
```

Comment:
overkill. you can put this inline in the only place you call this method. there is no reason to thing the implicit addition of these two addresses will ever be relevant outside of the single place you use it.. and as a result you can dis-introduce the gas recorder to this class...

Reply:
I respectfully disagree :) 
1. I like this separation - the participant is aware of its internal addresses.
2. It's more readable IMO.
3. Participant is already well aware of many driver internals (we could have just kept a reference to the driver instead of collecting internals). It's an "inner class" of the driver.
4. Here are a few other reasons to use it - if we want a different reporting format (let's say CSV), or if we want to test a scenario where the sum of gas used across a group of participants is what we measure.